docs/insights/api.md

```diff
@@ -0,0 +1,511 @@
+---
+title: API
+description: The Insights API
+position: 160
+---
+
+Insights is still in development.  You can use this API, however, please keep in mind that it could change at any time.
+
+# Insights Reports API
+
+## Get a list of Insights reports
+GET: api/{spaceId}/insights/reports
```

Comment:
I don't know if the leading slash should be added. I checked my API, and it showed this:

![image](https://user-images.githubusercontent.com/1418993/193311186-0526c799-354a-462d-9914-05a18a4e1818.png)

```suggestion
GET: `/api/{spaceId}/insights/reports`
```